spec/unit/views/components/paginated_collection_spec.rb

```diff
@@ -216,5 +216,20 @@ def paginated_collection(*args)
       end
     end
 
+    context "when specifying per_page: array option" do
+      let(:collection) do
+        posts = 10.times.map{ Post.new }
+        Kaminari.paginate_array(posts).page(1).per(5)
+      end
+
+      let(:pagination) { paginated_collection(collection, per_page: [1,2,3]) }
+      let(:pagination_html) { pagination.find_by_class('pagination_per_page').first }
+
+      it "should render per_page select tag" do
+        expect(pagination_html.content).to match(/Per page:/)
+        expect(Capybara.string(pagination_html.to_s)).to have_css('select option', count: 3)
```

Comment:
Line is too long. [92/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.